python/scope.py

```diff
@@ -517,8 +534,10 @@ def type_check_expr(self, expr):
 			for i, e in enumerate(rest):
 				if e != first:
 					self.errors.append(TypeCheckError(expr.children[i+1], "The list item #%s's type is %s while the first item's type is %s" % (i + 2, e, first)))
-			
+
```

Comment:
very cool github

Reply:
lol my editor is set to automatically remove trailing whitespace